src/components/Board.js

```diff
@@ -9,12 +9,16 @@ const generateSquareComponents = (squares, onClickCallback) => {
   // squares is a 2D Array, but 
   //  you need to return a 1D array
   //  of square components
+  const flatArray = [].concat.apply([], squares);//we made a 1d array
+  const squaresList = flatArray.map(square => {
+    return <Square value={square.value} id={square.id} onClickCallback={onClickCallback}/>
```

Comment:
You should include a `key={someUniqueValue}` attribute in any React component returned from `.map`.    That's why you're getting the warning the tests show.

```suggestion
    return <Square value={square.value} id={square.id} onClickCallback={onClickCallback} key={square.id}  />
```